tools/install-powershell.ps1

```diff
@@ -121,15 +148,19 @@ try {
         $packagePath = Join-Path -Path $tempDir -ChildPath $packageName
         Invoke-WebRequest -Uri $downloadURL -OutFile $packagePath
 
+        New-Item -ItemType Directory -Path "$Destination.new" > $null
```

Comment:
Please use the `$tempDir` for anything that is temporary, it's cleaner and also prevent the corner case taht `$Destination.new` already exists.
For example
```
$packageContent = Join-Path -Path $tempDir -ChildPath content
New-Item -ItemType Directory -Path $packageContent > $null
...
if ($IsWinEnv) {
    Expand-Archive ... -DestinationPath $packageContent
} else {
    tar zxf  ... -C $packageContent
}

Reply:
Will update